File: atividade-4/051_paridade/051_paridade.hs
module Main where

-- Faça uma função que verifica se o total de Trues é ímpar então retorne True e do contrário False

paridade :: [Bool] -> Bool
paridade xs = length [x | x <- xs, x == True] `mod` 2 /= 0

main = do
  a <- readLn :: IO [Bool]
  print $ paridade a

File: atividade-3/modulo-4/051_paridade/051_paridade.js
//Faça uma função que verifica se o total de Trues é ímpar então retorne True e do contrário False

const list = [true, true, false, true, false];
// const list = [true, true, false];

const paridade = (xs) => xs.filter((x) => Boolean(x)).length % 2 !== 0;

console.log(paridade(list));
